blocks/library/image/index.js

```diff
@@ -72,6 +75,23 @@ registerBlock( 'core/image', {
 	edit( { attributes, setAttributes, focus, setFocus } ) {
 		const { url, alt, caption } = attributes;
 
+		if ( ! url ) {
+			return (
```

Comment:
It means when it acts as a placeholder, when no content is yet set. This is going to be a prevalent case for templates down the road.